scanner/src/main/kotlin/scanners/Askalono.kt

```diff
@@ -91,17 +88,17 @@ class Askalono(name: String, config: ScannerConfiguration) : LocalScanner(name,
                 log.info { "Retrieved $scannerName from local cache." }
             }
 
-            val scannerDir = createTempDir(ORT_NAME, "$scannerName-$scannerVersion").apply { deleteOnExit() }
+            val unpackDir = createTempDir(ORT_NAME, "$scannerName-$scannerVersion").apply { deleteOnExit() }
 
-            val scannerFile = scannerDir.resolve(scannerExe)
-            scannerFile.sink().buffer().use { it.writeAll(body.source()) }
+            log.info { "Unpacking '$archive' to '$unpackDir'... " }
+            body.byteStream().unpack(archive, unpackDir)
 
             if (!Os.isWindows) {
-                // Ensure the executable Unix mode bit to be set.
-                scannerFile.setExecutable(true)
+                // The Linux version is distributed as a ZIP, but without having the Unix executable mode bits stored.
```

Comment:
In line 66-70 we distinguish four cases for the OS while here `!Os.isWindows == true` seems to imply `isLinux == true`.
How about mentioning macOs in addition to Linux?

Reply:
That was actually a left-over from copying the code from

https://github.com/oss-review-toolkit/ort/blob/462fcc736fbe62e989b97fb318bac7ee7011f221/scanner/src/main/kotlin/scanners/BoyterLc.kt#L87-L109

and can be removed here, thanks for noticing.

Reply:
Thanks!